app/src/main/res/layout/activity_main.xml

```diff
@@ -4,15 +4,7 @@
     xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
-    tools:context=".MainActivity">
-
-    <TextView
-        android:layout_width="wrap_content"
-        android:layout_height="wrap_content"
-        android:text="Hello World!"
-        app:layout_constraintBottom_toBottomOf="parent"
-        app:layout_constraintEnd_toEndOf="parent"
-        app:layout_constraintStart_toStartOf="parent"
-        app:layout_constraintTop_toTopOf="parent" />
-
-</androidx.constraintlayout.widget.ConstraintLayout>
\ No newline at end of file
+    tools:context=".MainActivity"
+    android:background="@drawable/ic_launcher_background"
```

Comment:
let's try to add a nice image for our splash screen instead of this android default logo.
(in the future we might change the app icon as well). Also lets not set it full screen cuz it doesn't look good on all devices, please add an ImageView in the center of the screen.